extensions/widgets/graph/graph.lcb

```diff
@@ -1053,6 +1098,29 @@ private handler drawGraph() returns nothing
 		end if
 		set the paint of this canvas to solid paint with tColour
 		
+		variable tFilledMarkers as Boolean
+		variable tMarkerPath as Path
+		
+		if mVerticesShow is true then
+			if mMarkerStyles[tGraphLine - 1] contains "circle" then
+				put circle path centered at point [0,0] with radius tMarkerSize into tMarkerPath
+			
+				put mMarkerStyles[tGraphLine - 1] contains "filled" into tFilledMarkers
+			else if mMarkerStyles[tGraphLine - 1] contains "square" or mMarkerStyles[tGraphLine - 1] contains "diamond" then
+				put rectangle path of rectangle [-tMarkerSize,-tMarkerSize,tMarkerSize,tMarkerSize] into tMarkerPath
+				
+				if mMarkerStyles[tGraphLine - 1] contains "diamond" then
+					rotate tMarkerPath by 45
+				end if
+				
+				put mMarkerStyles[tGraphLine - 1] contains "filled" into tFilledMarkers
+			else
+				put path iconSVGPathFromName(mMarkerStyles[tGraphLine - 1]) into tMarkerPath
+				put true into tFilledMarkers
+				constrainPathToRect(rectangle [-tMarkerSize,-tMarkerSize,tMarkerSize,tMarkerSize],tMarkerPath)
+			end if
```

Comment:
I think this code might be easier to read if you put `mMarkerStyles[tGraphLine - 1]` into a local variable within the `if mVerticesShow is true then...` block.